OracleHTTPServer/COPYRIGHT

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2001, 2019, Oracle Corporation.  All rights reserved.
+Copyright (c) 2001, 2024, Oracle Corporation.  All rights reserved.
```

Comment:
Latest copyright content looks like this:

Copyright (c) 2020, 2024, Oracle and/or its affiliates.
Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.